### DISCUSSION TITLE
Will a github username change also change a github sites URL?

### DISCUSSION THREAD

**Discussion by scuffedplus** [community/community#40874] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

In the title,
If you were to change your github username, would it also change the URL to a sites page? Given the page name is linked to your username. Or, will the site remain named after the previous username. Detached from giving the site a custom URL.

**Comment by DSDmark** [comment, marked as answer]
Hi @scuffedplus,

I guess, if you do that, everything will really get messed up. The pattern of GitHub page is like ```https://{GITHUB-USERNAME}.github.io/{YOUR-PEPO-NAME}```. 

> So, think before you will do.

There is clearly mention by **GitHub**.

![image](https://user-images.githubusercontent.com/85744816/205800977-3f370a3b-4a75-4ea4-828f-0535a4121b8b.png)


For more details, you could read this :- [**GitHub Docs**](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username)

**Comment by ahfilho** [comment]
Bom, a seguir a lógica da sua pergunta, sim, irá alterar tudo o que utiliza o nome.
